### DISCUSSION TITLE
Pybamm EIS and OKane2022 Degradation model

### DISCUSSION THREAD

**Discussion by Akila1993** [pybamm-team/PyBaMM#3536] [Q&A]
I am writing to you regarding the discussion I have opened [here](https://github.com/pybamm-team/PyBaMM/discussions/2974#discussioncomment-7584156) regarding the difference between time domain and frequency domain EIS profiles. 

According to the suggestion by Dr. Ferran when we reduce the amplitude of input current the difference between two profiles is reducing. According to Ferran it is because pybamm-eis uses a linearised version of the model so it is only an approximation. 

Could someone please elaborate this behaviour further like how (what equations) it is linearised or if we could use the nonlinear model / or linear model to match the Okane 2022 parameter set and the model. 

Thank you.

**Comment by brosaplanella** [comment]
Standard EIS uses small current variations and hence it deals with a linear approximation to the model. This is a very standard technique and there is a lot of literature on the topic, e.g. [this website](https://www.gamry.com/application-notes/EIS/basics-of-electrochemical-impedance-spectroscopy/), but a quick Google search throws a lot more.

Nonlinear EIS is also possible (see [this article by Kirk et al](https://iopscience.iop.org/article/10.1149/1945-7111/acada7/meta)) but this is a brand-new technique.
